convert_wide_to_long_format/convert_wide_to_long_format.py

```diff
@@ -0,0 +1,74 @@
+import os
+import argparse
+import gzip
+import re
+
+COLUMN= 'COLUMN';
+VALUE= 'VALUE';
+
+def map_row_with_header_index(row, header_index, output_file):
+    for header_row in header_index:
+        new_row_dict = {
+            key: row[idx] if isinstance(idx, int) and 0 <= idx < len(row) else idx
+            for key, idx in header_row.items()
+        }
+        output_file.write("\t".join(str(v) for v in new_row_dict.values()) + "\n")
+
+def run(input_file, output_file, custom_columns):
+    # write the array of objects to a csv file
+    if os.path.exists(output_file):
+        os.remove(output_file)  # Overwrite the file with an empty DataFrame
+        print(f"File '{output_file}' is already exist, so it is cleared first to remove the old contents")
```

Comment:
This is not necessary, since on line 24 you open the file in write mode, so the file is emptied if it already exists.